server/index.js

```diff
@@ -8,23 +8,7 @@ app.use(express.urlencoded({extended: true}));
 app.use(cors());
 
 require('./config/mongoose.config');
-
 require('./routes/project.route')(app);
 
```

Comment:
Hi Sara, you created a user route but its not being used by the backend app. I noticed this while I was testing my sign-up component to make sure it was actually posting to the db. 

```suggestion
require('./routes/user.route')(app);
```

Reply:
Oh yea, I fixed that. Already committed